src/lib/airtable/tables/requests.js

```diff
@@ -1,6 +1,6 @@
 const { merge } = require("lodash");
 const _ = require("lodash");
-const { airbase } = require("~airtable/bases");
+const { airbase } = require("../bases");
```

Comment:
For some reason I had to make this a relative path. The tilde path threw an error.

Reply:
very weird! do you remember the error? this should be currently in prod right?

Reply:
Yeah, super weird! The version with the tilde path is in prod, which proves that it works. Maybe this is just user error on my part.

Here's the error: `Module not found: Can't resolve '~airtable/bases' in '/Users/emmasmith/projects/mutual-aid-app/src/lib/airtable/tables'`

I see this when I run `npm run local:react`. Also when I try to build for prod using `npm run build`. The Express app runs just fine. It seems that the React side of things is not respecting the `moduleAliases`. 

Clues, advice, guidance most appreciated :)